rskj-core/src/main/java/org/ethereum/vm/program/invoke/ProgramInvokeImpl.java

```diff
@@ -313,75 +313,6 @@ public int getCallDeep() {
         return this.callDeep;
     }
 
-    @Override
-    public boolean equals(Object o) {
```

Comment:
We've tested this method (with @ajlopez) by throwing an exception inside it, and commenting the rest of the code, there were no failing tests, so we decided to delete it to avoid unnecessary code.

Reply:
danger :P

Reply:
Now I've added the `hashcode` method instead of deleting the `equals`.